pkgs/development/ruby-modules/bundler-env/default.nix

```diff
@@ -46,11 +46,20 @@ let
 
   importedGemset = import gemset';
 
-  filteredGemset = (lib.filterAttrs (name: attrs:
-    if (builtins.hasAttr "groups" attrs)
-    then (builtins.any (gemGroup: builtins.any (group: group == gemGroup) groups) attrs.groups)
-    else true
-  ) importedGemset);
+  platformMatches = attrs: (
+  !(attrs ? "platforms") ||
+    builtins.any (platform:
+      platform.engine == ruby.rubyEngine &&
+        (!(platform ? "version") || platform.version == ruby.version.majMin)
```

Comment:
a note to myself: make sure these are the right version that are being matched. On MRI is't 1-1 but not on jruby for example.